account_move_line_sequence/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,2 @@
+This module adds sequence to account_move_line
```

Comment:
It's good to be explicit about the use case - it's only for visually reordering the journal items inside a journal entry.